# Pre-defined joint positions.
# pose entries should follow the pattern p0,p1...
joint_poses:
  p0:
    name: zero
    state:
      [0.0, 0.0, 0.0, 0.0,
      0.0, 0.0, 0.0, 0.0,
      0.0, 0.0, 0.0, 0.0,
      0.0, 0.0, 0.0, 0.0]
  p1:
    name: relax
    state:
      [0.254, 0.219, 0.712, 0.478,
      0.085, 0.145, 0.879, 0.330,
      -0.072, 0.220, 0.765, 0.610,
      1.325, 0.000, 0.122, 0.386]
  p2:
    name: parallel
    state:
      [0.012, 0.913, 0.426, 0.748,
      0.028, 0.843, 0.621, 0.715,
      0.023, 0.760, 0.684, 0.587,
      1.557, 0.094, -0.189, 1.060]
  p3:
    name: point1
    state:
      [0.06, 0.50, 0.13, -0.14,
      0.0, 1.32, 1.72, 0.0,
      0.0, 1.32, 1.72, 0.0,
      0.72, 0.01, 0.40, 1.5]
  p4:
    name: grip
    state:
      [0.046, 0.974, 0.505, 0.836,
      0.058, 0.842, 0.909, 0.419,
      0.089, 0.887, 0.813, 0.327,
      1.550, 0.307, -0.180, 1.300]
